apps/api/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9
+
+COPY requirements.txt requirements.txt
+
+RUN apt-get update && apt-get install -y libxml2-dev libxmlsec1-dev libxmlsec1-openssl
+RUN pip install --no-cache-dir -r requirements.txt "uvicorn[standard]"
+
+EXPOSE 8000
```

Comment:
Question: do we need to expose the port here? I thought the `ports` part of the compose file would handle it.

Reply:
No, I was using this during a debugging process, since I was trying to run just this container by itself. I'll remove this.